request-server_test.go

```diff
@@ -345,6 +345,40 @@ func TestRequestFstat(t *testing.T) {
 	checkRequestServerAllocator(t, p)
 }
 
+func TestRequestFsetstat(t *testing.T) {
+	p := clientRequestServerPair(t)
+	defer p.Close()
+	_, err := putTestFile(p.cli, "/foo", "hello")
+	assert.Nil(t, err)
```

Comment:
`require.NoError`. If this errors, we want to stop right here, and now, because nothing else will work.